presto-docs/src/main/sphinx/rest/query.rst

```diff
@@ -26,7 +26,7 @@ on a Presto installation.
 
    **Example response:**
 
-   .. code-block:: json
+   .. code-block:: text
```

Comment:
Here I think the issue is because of the below JSON block, I tried making this change now I don't warning. Could you please check?
![image](https://github.com/prestodb/presto/assets/7887476/3381754e-4e96-4c0c-86b6-1912b7548a79)